addons/l10n_my_edi_extended/tests/test_new_submissions_flow.py

```diff
@@ -327,6 +327,31 @@ def test_11_qr_code_generation(self):
         qr_data_uri = self.basic_invoice._generate_myinvois_qr_code()
         self.assertTrue(qr_data_uri)
 
+    def test_12_multiple_moves_with_one_failed_submission(self):
+        """Test that when submitting multiple invoices, only one move fails due to incorrect document,
+        while others succeed."""
+        self.submission_count = 0
+        invoice_vals = []
+        for i in range(1, 10):
+            invoice_vals.append({
+                'move_type': 'out_invoice',
+                'partner_id': self.partner_a.id,
+                'invoice_line_ids': [
+                    Command.create({'product_id': self.product_a.id}),
+                ],
+            })
+
+        self.submission_invoice = self.env['account.move'].create(invoice_vals)
+        self.submission_invoice.action_post()
+        self.submission_invoice |= self.basic_invoice
+
+        with patch(CONTACT_PROXY_METHOD, new=self._test_12_mock), \
+             patch('odoo.addons.l10n_my_edi.models.account_move.SUBMISSION_MAX_SIZE', 2):
+            self.submission_invoice.action_l10n_my_edi_send_invoice()
+
+        # we have 10 invoices, with a max size of 2 we expect 5 different submissions.
+        self.assertEqual(self.submission_count, 5)
```

Comment:
It would be more relevant to assert the state of the invoices and to validate that one submission failed (but no traceback was raised) while the other worked